elastic/logstash
# plugin manager uninstall plugin regardless of working directory

## Release notes


Fix: allow plugin manager to remove plugin regardless of the current working directory

## What does this PR do?



change the removal procedure from relative path to absolute path

## Why is it important/What is the impact to the user?



This PR fixes an issue that was preventing user uninstalls plugin from the plugin manager when user is not in $LOGSTASH_HOME directory

## Author's Checklist


- [x] tested in macOS

## How to test this PR locally



Assuming your Logstash is not installed in home directory
In mac
run uninstall command outside $LOGSTASH_HOME
> cd ~
> $LOGSTASH_HOME/bin/logstash-plugin remove logstash-filter-kv

terminal should show "Successfully removed logstash-filter-kv"

## Related issues


Closes #10781
